x-pack/filebeat/module/rabbitmq/log/test/test.log-expected.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "@timestamp": "2019-04-03T11:13:15.076-02:00",
+        "@timestamp": "2023-01-03T05:20:19.811-02:00",
```

Comment:
There is no difference in the @timestamp before vs after?

Reply:
The @timestamp field is parsed from timestamp field, the actual change is in timestamp field.